nvtabular/workflow/node.py

```diff
@@ -18,6 +18,8 @@
 
 from nvtabular.columns import ColumnSelector, Schema
 from nvtabular.ops import LambdaOp, Operator, internal
+from nvtabular.ops.internal.concat_columns import ConcatColumns
+from nvtabular.ops.internal.subset_columns import SubsetColumns
```

Comment:
We should be able to import these in one line

Reply:
like `from nvtabular.ops.internal import ConcatColumns, SubsetColumns`